docs/development/prerequisites.md

```diff
@@ -37,30 +37,30 @@ Alternatively, **Windows** users may follow these steps:
 #### [Helm](https://github.com/kubernetes/helm/blob/master/docs/install.md)
 
 Alternatively, **Windows** users may follow [these steps](https://medium.com/@JockDaRock/take-the-helm-with-kubernetes-on-windows-c2cd4373104b):
-    
+
 
 Afterwards, start Minikube and initialize Tiller.
 
-    ```bash
-    minikube start --extra-config=apiserver.Authorization.Mode=RBAC
+```bash
```

Comment:
Shouldn't **console** be used here (instead of **bash**) and in the rest of the file - just like it's done in other files?